scripts/ci/pre_commit/pre_commit_compile_www_assets.py

```diff
@@ -17,17 +17,31 @@
 # under the License.
 import os
 import subprocess
+import sys
 from pathlib import Path
 
+sys.path.insert(0, str(Path(__file__).parent.resolve()))  # make sure common_precommit_utils is imported
+from common_precommit_utils import get_directory_hash  # isort: skip # noqa
```

Comment:
I cannot use the nice tool to calculate hash of directories that I use in "doc" (checksumdir), because the scripts are using  "node" environment in pre-commit, and we cannot add extra python dependencies, so I have to rely on stdlib. The "get_directory_hash" is rather small and simple.